base/routes.js

```diff
@@ -48,17 +62,48 @@ export const trees = mdRoutes.reduce((out, { name, path, children = [], data = [
   return out
 }, {})
 
-const routes = Object.keys(trees).reduce((out, key) => {
-  const { tree } = trees[key]
-  const reduced = tree.reduce((acc, cur) => acc.concat(flatten(reduction(cur))), [])
-  const final = [...reduced, reduced[0] && { path: key, redirect: reduced[0].path }]
-    .filter(d => d)
-    .sort((a, b) => (b.redirect ? -1 : a.redirect ? 1 : 0))
+const routes = Object.keys(trees)
+  .reduce((out, key) => {
```

Comment:
Nit: I'm sure this usage of reduce is very compact but I do find it quite hard to read compared to for loops. But I suppose you are just reformatting existing code.

Reply:
lol, it's less compact that what it's replacing. but i see your point